.changeset/ten-spiders-trade.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hyperlane-xyz/cli': patch
+---
+
+Added ZKSync signer support using zksync-ethers package
```

Comment:
need to update the changeset and change to major

Reply:
This is related to this PR https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/4914/files

Reply:
gotcha, would need a new changeset for this PR then

Reply:
Since there are a lot of changes on `sdk` and `core` I propose we add new 'major' changeset for these too? @paulbalaji

Reply:
yup I'd agree